data/strings/types_strings.txt

```diff
@@ -16533,7 +16535,7 @@
     mr = पाण्याचा नळ
     nb = Vannkran
     nl = Waterkraan
-    pl = Woda pitna
+    pl = Kran z wodą
```

Comment:
What is selected for `man_made=water_tap amenity=drinking_water` ?

Reply:
In current version we have two "Woda pitna" (for amenity.drinking_water & man_made.water_tap), so I changed the second one for "Kran z wodą" (water from it have not be for drinking)

Reply:
My point is that `man_made=water_tap amenity=drinking_water` should show up as "Woda pitna" (though that may require tweaks beyond translations)

Reply:
Yes, understood, but you are thinking on tag combination. Types_strings correspond to adding a new POI via OM app, and doing this we can just use one, single tag (without combination). So we have to be clear with distinction for drinking water (-> amenity=drinking_water) and "just water" (-> man_made=water_tap), when we don't know nothing about water quality.